projects/u-mill/white_mesa/leaching.py

```diff
@@ -166,7 +166,7 @@ def __init__(self):
         self.five_tau_preleach_dissolution = 4.5 * unit.hour # 5 * relaxation time
 
         self.acidleach_tank_vol = 7 * math.pi*(7.6/2)**2 * unit.meter**2 * 8.2 * unit.meter
-        print(self.acidleach_tank_vol)
```

Comment:
Why are you changing leaching?

Reply:
This must have reverted back by accident. I had to redownload the entire repository due to the extent of conflicts in my version of the code. This can be put back.